### DISCUSSION TITLE
PGSync in Single pass mode

### DISCUSSION THREAD

**Discussion by ravin-accelq** [toluaina/pgsync#419] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/40689807/220824251-20c80a56-a6b9-42bf-9c03-1bd17f29e9cf.png)

In the above image it says omit --daemon to run it in single pass. Does the single pass here takes into account the last transaction ID and continue pulling from that checkpoint?

**Comment by toluaina** [comment, marked as answer]
Yes that is correct. Daemon or non-daemon mode will maintain the last state of the transaction id
